cmd/cigocacher/disk_windows.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
```

Comment:
this does not need a windows build tag?

Reply:
"Build constraints may also be part of a file's name (for example, source_windows.go will only be included if the target operating system is windows).

See 'go help buildconstraint' (https://pkg.go.dev/cmd/go#hdr-Build_constraints) for details."

-- https://pkg.go.dev/go/build